src/main/java/de/mediathekview/mserver/crawler/srf/tasks/SrfFilmDetailTask.java

```diff
@@ -0,0 +1,81 @@
+package de.mediathekview.mserver.crawler.srf.tasks;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.reflect.TypeToken;
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractRestTask;
+import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.srf.SrfConstants;
+import de.mediathekview.mserver.crawler.srf.parser.SrfFilmJsonDeserializer;
+import java.lang.reflect.Type;
+import java.net.URI;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class SrfFilmDetailTask extends AbstractRestTask<Film, CrawlerUrlDTO> {
+
+  private static final Logger LOG = LogManager.getLogger(SrfFilmDetailTask.class);
+  
+  public SrfFilmDetailTask(AbstractCrawler aCrawler, ConcurrentLinkedQueue<CrawlerUrlDTO> aURLsToCrawl)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aURLsToCrawl" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)